src/cow-react/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList.ts

```diff
@@ -1,11 +1,30 @@
 import { useWeb3React } from '@web3-react/core'
 import { useOrders } from 'state/orders/hooks'
-import { useMemo } from 'react'
+import { useCallback, useMemo } from 'react'
 import { Order, OrderStatus } from 'state/orders/actions'
+import BigNumber from 'bignumber.js'
```

Comment:
Can we use `@ethersproject/bignumber` or `JSBI` instead? I think we don't need several implementation of bignumber

Reply:
Not sure about JSBI, because it will complain about the decimals and we have decimals in some of these values. Its not ideal that we are mixing these types but I suggest we fix this in the future.